R/enhanceFeatures.R

```diff
@@ -12,6 +12,8 @@
 #'   attached to \code{sce.ref}. Must have columns corresponding to the spots in
 #'   \code{sce.ref}. Overrides \code{assay.type} and \code{altExp.type} if
 #'   specified.
+#' @param tune.nrounds logical. If \code{TRUE}, tune the nrounds parameter for xgboost.
```

Comment:
it might be helpful to include notes on the impact on runtime and what the tuning process entails/yields